src/style.css

```diff
@@ -1 +1,60 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  color: #3d4e61;
+}
```

Comment:
You've added a `color` property in the `@font-face` rule. This property is not valid within `@font-face` and will be ignored. You should move it to the selector where you want to apply this text color.